.github/workflows/publish.yml

```diff
@@ -17,8 +17,6 @@ jobs:
           registry-url: https://registry.npmjs.org/
       - name: NPM Install
         run: npm install
-      - name: NPM run build
```

Comment:
npm install runs `prepare` npm run script which is running `npm run build`.

Reply:
It looks like npm `publish` also calls `prepare` script. Is there any way to not have repeat the build twice here?

What if I just want to do a npm install? I have to wait for the prepare script to finish also?

Reply:
Yes multiple npm scripts actually call `prepare` script, and hence I mentioned this would not be a good way. But I could only found this way to make it work upon installation from GitHub directly currently.